src/os/host-repl.r

```diff
@@ -279,7 +279,7 @@ host-repl: function [
                 ; and needs to be closed.  Invert the symbol.
                 ;
                 unclosed: switch error/arg1 [
-                    "}" ["{"]
+                    "}" ["^{"]
```

Comment:
At the moment I need this because otherwise it screws up my Vim syntax highlighting :(

I will switch it back to `"{"` once i've fixed Vim rebol syntax or when finished with REPL fixes.